app/actions/remote/channel.ts

```diff
@@ -49,6 +51,98 @@ export type MyChannelsRequest = {
     error?: unknown;
 }
 
```

Comment:
Be wary of extra `await`s. They are generally not needed unless for specific use cases.  For example, perhaps we don't need to await for `updateLocalUser` in the function `getMemberInChannel` as it doesn't create any visual update for the user to interact with.  Certain things can happen in the background at their own pace without blocking.

Reply:
This was helpful. Thanks for the suggestion!